MapboxCoreNavigation/NavigationRouteOptions.swift

```diff
@@ -22,7 +22,7 @@ open class NavigationRouteOptions: RouteOptions {
         }, profileIdentifier: profileIdentifier)
         includesSteps = true
         routeShapeResolution = .full
-        attributeOptions = [.congestionLevel, .expectedTravelTime]
+        attributeOptions = [.congestionLevel, .expectedTravelTime, .speed]
```

Comment:
This is a large increase in the response payload for a feature that's normally only used in debugging. The caller should take care of adding this option if simulation is enabled.

/cc @frederoni @bsudekum

Reply:
Besides, we can derive the speed from the distance between two nodes and the expected travel time along that segment – no additional data needed.